.gitignore

```diff
@@ -1,15 +1,21 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
-/node_modules
+node_modules
+ammo-fe/node_modules
+ammo-be/node_modules
+
 /.pnp
 .pnp.js
 
 # testing
-/coverage
+ammo-be/coverage
```

Comment:
To prevent duplicates between ammo-be & ammo-fe, i suggest using '**/coverage' (same goes for other folders)